lib/core/menu.rb

```diff
@@ -0,0 +1,39 @@
+require 'UI/UI'
+require 'core/players/player_factory'
+require 'core/players/human'
+require 'core/players/easy_computer'
+require 'core/flow'
+
+class Menu
+  attr_reader :ui, :mode, :players, :flow
+
+  def initialize
+    @ui = UI.new
```

Comment:
How are you testing this class? I would assume that in order to test it you would have to inject the UI, otherwise mocking the IO would be complicated.